python313Packages.reflex: 0.7.4a0 -> 0.7.5

pbsds opened the pull request:
Changelog: https://github.com/reflex-dev/reflex/releases/tag/v0.7.4
Changelog: https://github.com/reflex-dev/reflex-chakra/releases/tag/v0.7.0


<!--
^ Please summarise the changes you have done and explain why they are necessary here ^

For package updates please link to a changelog or describe changes, this helps your fellow maintainers discover breaking updates.
For new packages please briefly describe the package or provide a link to its homepage.
-->

## Things done

<!-- Please check what applies. Note that these are not hard requirements but merely serve as information for reviewers. -->

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- For non-Linux: Is sandboxing enabled in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
  - [ ] `sandbox = relaxed`
  - [ ] `sandbox = true`
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#linking-nixos-module-tests-to-a-package) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [25.05 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2505.section.md) (or backporting [24.11](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2411.section.md) and [25.05](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2505.section.md) Release notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

<!--
To help with the large amounts of pull requests, we would appreciate your
reviews of other pull requests, especially simple package updates. Just leave a
comment describing what you have tested in the relevant package/service.
Reviewing helps to reduce the average time-to-merge for everyone.
Thanks a lot if you do!

List of open PRs: https://github.com/NixOS/nixpkgs/pulls
Reviewing guidelines: https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#reviewing-contributions
-->

---

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

pbsds committed:
python313Packages.reflex: 0.7.4a0 -> 0.7.4

Changelog: https://github.com/reflex-dev/reflex/releases/tag/v0.7.4

pbsds committed:
python313Packages.reflex-chakra: 0.6.3 -> 0.7.0

Changelog: https://github.com/reflex-dev/reflex-chakra/releases/tag/v0.7.0

pbsds committed:
granian: 1.7.0 -> 2.2.0

pbsds committed:
granian: move to python3Packages

pbsds commented:
## `nixpkgs-review` result

Generated using [`nixpkgs-review`](https://github.com/Mic92/nixpkgs-review).

Command: `nixpkgs-review pr 392890`

---
### `x86_64-linux`
<details>
  <summary>:white_check_mark: 12 packages built:</summary>
  <ul>
    <li>granian (python312Packages.granian)</li>
    <li>granian.dist (python312Packages.granian.dist)</li>
    <li>python312Packages.reflex</li>
    <li>python312Packages.reflex-chakra</li>
    <li>python312Packages.reflex-chakra.dist</li>
    <li>python312Packages.reflex.dist</li>
    <li>python313Packages.granian</li>
    <li>python313Packages.granian.dist</li>
    <li>python313Packages.reflex</li>
    <li>python313Packages.reflex-chakra</li>
    <li>python313Packages.reflex-chakra.dist</li>
    <li>python313Packages.reflex.dist</li>
  </ul>
</details>

---
### `aarch64-linux`
<details>
  <summary>:white_check_mark: 12 packages built:</summary>
  <ul>
    <li>granian (python312Packages.granian)</li>
    <li>granian.dist (python312Packages.granian.dist)</li>
    <li>python312Packages.reflex</li>
    <li>python312Packages.reflex-chakra</li>
    <li>python312Packages.reflex-chakra.dist</li>
    <li>python312Packages.reflex.dist</li>
    <li>python313Packages.granian</li>
    <li>python313Packages.granian.dist</li>
    <li>python313Packages.reflex</li>
    <li>python313Packages.reflex-chakra</li>
    <li>python313Packages.reflex-chakra.dist</li>
    <li>python313Packages.reflex.dist</li>
  </ul>
</details>

---
### `x86_64-darwin`
<details>
  <summary>:white_check_mark: 12 packages built:</summary>
  <ul>
    <li>granian (python312Packages.granian)</li>
    <li>granian.dist (python312Packages.granian.dist)</li>
    <li>python312Packages.reflex</li>
    <li>python312Packages.reflex-chakra</li>
    <li>python312Packages.reflex-chakra.dist</li>
    <li>python312Packages.reflex.dist</li>
    <li>python313Packages.granian</li>
    <li>python313Packages.granian.dist</li>
    <li>python313Packages.reflex</li>
    <li>python313Packages.reflex-chakra</li>
    <li>python313Packages.reflex-chakra.dist</li>
    <li>python313Packages.reflex.dist</li>
  </ul>
</details>

---
### `aarch64-darwin`
<details>
  <summary>:white_check_mark: 12 packages built:</summary>
  <ul>
    <li>granian (python312Packages.granian)</li>
    <li>granian.dist (python312Packages.granian.dist)</li>
    <li>python312Packages.reflex</li>
    <li>python312Packages.reflex-chakra</li>
    <li>python312Packages.reflex-chakra.dist</li>
    <li>python312Packages.reflex.dist</li>
    <li>python313Packages.granian</li>
    <li>python313Packages.granian.dist</li>
    <li>python313Packages.reflex</li>
    <li>python313Packages.reflex-chakra</li>
    <li>python313Packages.reflex-chakra.dist</li>
    <li>python313Packages.reflex.dist</li>
  </ul>
</details>

pbsds committed:
python313Packages.granian: enable checks, add pbsds as maintainer

GaetanLepage reviewed (commented):
This looks very good!
A few minor comments:

pbsds commented on pkgs/development/python-modules/granian/default.nix:
```diff
@@ -0,0 +1,85 @@
+
+  pythonImportCheck = [ "granian" ];
+
+  passthru.updateScript = nix-update-script { };
+
```
It is not

https://github.com/NixOS/nixpkgs/blob/65eda8168822f411fdda8fd34948ffcb65df16cd/pkgs/development/interpreters/python/mk-python-derivation.nix#L409-L416

pbsds committed:
python313Packages.reflex: 0.7.4 -> 0.7.5

Diff: https://github.com/reflex-dev/reflex/compare/refs/tags/v0.7.4...v0.7.5

Changelog: https://github.com/reflex-dev/reflex/releases/tag/v0.7.5

GaetanLepage reviewed (approved):
LGTM!
